content/content-bundles/resources/content.properties

```diff
@@ -665,3 +665,5 @@ createUrls.requiredfield.name=Website Name is a required field. Please provide a
 
 access.hidden=(Hidden)
 resource.is.hidden=This resource/collection is hidden
+
+resource.is.secured.access=The secured access is enabled for this file, it can be accessed under special conditions like an assessment.
```

Comment:
This resource is secured and can only be accessed under specific conditions.

Reply:
Done